### PR TITLE
Show API docs when user isn't logged in

### PR DESCRIPTION
**Pull request by rowanseymour**

### CODE REVIEW

File: temba/api/tests/test_v1.py [hunk 1]
```diff
@@ -173,7 +173,7 @@ def test_api_root(self):
 
         # browse as HTML anonymously
         response = self.fetchHTML(url)
-        self.assertContains(response, "RapidPro API", status_code=403)  # still shows docs
+        self.assertContains(response, "We provides a simple REST API", status_code=403)  # still shows docs
```
Comment (ericnewcomer) [inline temba/api/tests/test_v1.py]: Should this still be a 403?
"We provides" ?

Comment (rowanseymour) [inline temba/api/tests/test_v1.py]: Doh can't believe I didn't notice that spelling mistake. Hmm it's certainly a bit weird to return a forbidden code with regular content. I think a lot of DRF users are rendering forms on their endpoint documentation pages which one wouldn't want displayed for unauthenticated users. But then the 403 keeps the status consistent across HTML/XML/JSON formats...

**Comment by nicpottier** [comment]
What status we do return when the user is logged in and hits that page? Is that a 403 as well? We aren't exactly consistent across html/json for GETs as it is.

I don't actually feel strongly about showing content with 403s, I'm fine with that, but I think it should be the same response code for .html GETs. (though I could easily be convinced otherwise if that's a pain)

**Comment by rowanseymour** [comment]
Not sure I follow. All endpoints return 403 if not authenticated. With no suffix or .api suffix, the endpoints return 200 + documentation if authenticated. With the .json or .xml suffixes then it's whatever the view class returns. 

I'm fine if we return 403 + docs for unauthenticated requests (i.e. this pr).

**Comment by nicpottier** [comment]
Ah ok, ya I was wondering if we had any plain GETs (with no parameters) that ever returned non-200s for API endpoints, but I think they all just return the default queryset. So not relevant here.

403 with docs it is!

**Pull request merged**